tests/metactl/test-metactl.sh

```diff
@@ -16,3 +16,14 @@ cat $meta_json \
 ./target/debug/databend-metactl --export --raft-dir "$meta_dir" > $exported
 
 diff $meta_json $exported
+
+# test export from grpc
+chmod +x ./target/debug/databend-meta
+./target/debug/databend-meta --single &
+METASRV_PID=$!
+echo $METASRV_PID
+sleep 0.5
+
+./target/debug/databend-metactl --export
+
+kill $METASRV_PID
```

Comment:
The output of `./target/debug/databend-metactl --export` should be redirected to a file and the content of the file should be checked.

Reply:
I wonder if `databend-meta --single` exported data is identical, if so I'll use that.

Reply:
I can not tell if it is identical. But there should be at least a record like this:
```
["test-29000-raft_state",{"RaftStateKV":{"key":"Id","value":{"NodeId":0}}}]
```

Reply:
I find it's not identical. So just check `["raft_state",{"RaftStateKV":{"key":"Id","value":{"NodeId":0}}}]` in exported file.
Wondering if it's ok.